docs/epidemiology/covid_19_hospital_occupancy_simulator/simulation_flow.md

```diff
@@ -1,13 +1,11 @@
-# :material-chart-timeline-variant: Simulation Flow 
-
-The flow of a simulation, which subsequently generates a [simulation-based projection](./terminology.md#simulation-based-projection), can be summarized in the following sequential steps: 
+# :material-chart-timeline-variant: How Simulations Work
 
+There are three basic steps to run a simulation in CHOS.
```

Comment:
Replace simulation with this: 

`[simulation-based projection](./terminology.md#simulation-based-projection)`